src/scss/core/_base.scss

```diff
@@ -8,6 +8,7 @@ html {
 
 body {
   width: 100%;
+  overflow-x: clip;
```

Comment:
This is the change I added to remove that double vertical scrollbar. Please check that it doesn't affect other things (search modal, menu, other pages)